docs/help/account/billing.md

```diff
@@ -0,0 +1,44 @@
+# Billing
+
+The Billing section of your account provides access to the following items:
+
+- information about your current plan
+- your invoice history
+- your billing details
+
+Bump.sh offers monthly or annual billing. Payment is by credit card only; we do not offer any alternatives at this time. We use Stripe to handle all payments, so we never store your bank details.
+
+## Invoices
+
+### Edit Your Billing Information
+
+You can edit the following billing information so that it appears on your upcoming invoices:
+
+- Company name
+- Address
+- VAT Number
+
+:::info
+Edited information will only appear on future invoices; it is not possible to edit the details of an already created invoice.
+:::
+
+### Invoices
+
+Every month, we send your invoice in PDF format to the email address linked to your account. You can specify an alternative email address to which they will be sent, for your accounting department, for example.
+
+On this page, you will also find a history of your invoices, available in PDF.
+
+### Taxes
+
+We apply taxes in accordance with current regulations. Three scenarios are possible:
+
+- you or your company reside outside of Europe: we do not apply VAT.
```

Comment:
It should be only about company/business residency, and disregard the employee residency. I can work for a german company: if I purchase a tool for that company, VAT rules will apply according the the german location of the company.

If we believe there's a legitimate use case about individuals/freelancers purchasing the tool, and that we thereby cannot talk about a "company", I would switch the vocabulary to "billing address".

Example:
```
- your billing address is located outside of Europe: we do not apply VAT.
- your billing address is located in Europe: we apply VAT.
```

TBC, but I think there are actually 3 cases:
- out of Europe (no VAT)
- In France (charge 20%)
- not in France, but in Europe (No VAT)

Reply:
Louise know everything about VAT ;)

Reply:
👍

Reply:
> TBC, but I think there are actually 3 cases:
> 
> out of Europe (no VAT)
> In France (charge 20%)
> not in France, but in Europe (No VAT)

Nope. In Europe, we do not apply VAT if the user does provide a VAT number (there's a specific accounting rule on this, which is applied in our billing system).